sdk/batch/Microsoft.Azure.Batch/CHANGELOG.md

```diff
@@ -1,16 +1,24 @@
 # Release History
 
-## 15.2.0-beta.1 (Unreleased)
+## 15.3.0 (2022-01-01)
 
 ### Features Added
 
-### Breaking Changes
+- Added property `UploadHeaders` to `OutputFileBlobContainerDestination`.
+  - Allows users to set custom HTTP headers on resource file uploads.
+  - Array of type `HttpHeader` (also being added).
+- Added boolean property `AllowTaskPreemption` to `JobSpecification`, `CloudJob`, `JobAddParameter`, `JobPatchParameter`, `JobUpdateParameter`
+  - Mark Tasks as preemptible for higher priority Tasks (requires Comms-Enabled or Single Tenant Pool).
+- Replaced comment (title, description, etc.) references of "low-priority" with "Spot/Low-Priority", to reflect new service behavior.
+  - No API change required.
+  - Low-Priority Compute Nodes (VMs) will continue to be used for User Subscription pools (and only User Subscription pools), as before.
+  - Spot Compute Nodes (VMs) will now be used for Batch Managed (and only Batch Managed pools) pools.
 
-### Bugs Fixed
+## 15.2.0 (Unreleased)
```

Comment:
Let's just roll this change up to 15.3.0 if we never released 15.2.0 (and we can remove 15.2.0 from the release history)